maggy/install.sh

```
@@ -0,0 +1,88 @@
+  NEEDS_CONFIG=1
+else
+  echo "✓ Config already exists at $MAGGY_HOME/config.yaml (not overwritten)"
+  NEEDS_CONFIG=0
+fi
+
+# 5. Remember bootstrap location for iCPG integration
+BOOTSTRAP_MARKER="$HOME/.claude/.bootstrap-dir"
+if [ ! -f "$BOOTSTRAP_MARKER" ]; then
+  mkdir -p "$HOME/.claude"
+  # Maggy lives in <bootstrap>/maggy — one level up is bootstrap root
+  echo "$(cd "$HERE/.." && pwd)" > "$BOOTSTRAP_MARKER"
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)